crates/stages/stages/src/stages/execution.rs

```diff
@@ -270,6 +277,22 @@ where
             })?;
             execution_duration += execute_start.elapsed();
 
+            // Log execution throughput
+            if last_log_instant.elapsed() >= log_duration {
```

Comment:
I realize we could shave of a few internal Instant::now calls here by calculating elpased manually, but it's not really worth it.